Debian/8/input/oval/installed_OS_is_debian8.xml

```diff
@@ -10,21 +10,12 @@
       <reference source="JL" ref_id="DEBIAN8_20151210" ref_url="test_attestation" />
     </metadata>
     <criteria comment="current Debian version is Debian jessie" operator="AND">
-      <criterion comment="Installed operating system is part of the unix family" test_ref="test_unix_family" />
+      <extend_definition comment="Installed OS is part of the Unix family" definition_ref="installed_OS_is_part_of_Unix_family" />
```

Comment:
@redhatrises 
Because it's not truly addressing the real issue, but rather hiding it more deeply away by suppressing the corresponding warning message. Suppose the scenario there will be 20 (or more OS products in SSG), having corresponding ```installed_OS_is_name20.xml``` checks. Will we invent artificial names to these just to overcome the issue with duplicate ID?

While the above might work for some amount of time, the issue is more sneaky like immediately seems. Each new ID solves the problem with duplicate ID, but doesn't solve that same OVAL entities are used for the checks. When an issue is found in the implementation, we might fix some of them, but since the IDs would differ not expecting someone to inspect semantics of these other IDs named checks just to see if the change wouldn't apply there too.

Like for OVAL checks we have templates and checks, and if change is needed, we change the template and regenerate the resulting checks, IMHO the better approach is to reduce the shared OVAL entities into new definition, and reference it where necessary. In this approach (while might seem more time consuming for the implementation) if a OVAL check change is necessary, it will be enough to change the part in the extended definition and the change will reflect in all the OVALs, referencing / extending this definition.